alchemy/src/cloudflare/website.ts

```diff
@@ -70,20 +70,41 @@ export async function Website<B extends Bindings>(
   return alchemy.run(id, async () => {
     // building the site requires a wrangler.jsonc file to start
     // - so initialize an empty one if it doesn't exist
-
     const cwd = path.resolve(props.cwd || process.cwd());
-    const fileName =
-      typeof props.wrangler === "boolean"
-        ? "wrangler.jsonc"
-        : typeof props.wrangler === "string"
-          ? props.wrangler
-          : (props.wrangler?.path ?? "wrangler.jsonc");
-    const wranglerPath =
-      fileName && path.relative(cwd, path.join(cwd, fileName));
-    const wranglerMain =
-      typeof props.wrangler === "object"
-        ? (props.wrangler.main ?? props.main)
-        : props.main;
+
+    let fileName;
+    let wranglerPath;
+    let wranglerMain = path.join(cwd, props.main ?? "");
```

Comment:
Is `?? ""` right? In what situation would it be empty?

Reply:
props.main is optional